docs/release_process.rst

```diff
@@ -0,0 +1,78 @@
+ORA2 Release Processs
+=====================
+
+.. contents::
+
+Github Checklist
+----------------
+- create a PR on edx-ora2 containing the version bump. This needs to happen in  `setup.py <https://github.com/edx/edx-ora2/blob/4cc85e5a057fe8ea2d876e7c27344deb67df54d3/setup.py#L39>`_ and `package.json <https://github.com/edx/edx-ora2/blob/4cc85e5a057fe8ea2d876e7c27344deb67df54d3/package.json#L3>`_
+- get a green Travis build on said PR
+- merge to master
+- green build on master
+- create a release `tag on GitHub <https://github.com/edx/edx-ora2/releases>`_
+- ORA2 is a dependency in platform. If immediate release is needed, create PR to update `requirements files` in `edx-platform`. See `PR for reference <https://github.com/edx/edx-platform/pull/24830>`_ . Otherwise, a bot will automatically update the requirements in platform.
+- If manual testing of the changes against edx-platform is desired, create a sandbox.
+- get green build on the edx-platform PR, merge
+- manually test changes on stage
+
+Updating translations
+---------------------
+
+If you change any text that is ultimately wrapped by ugettext (in a python, JS, or HTML file), you'll have to update translations.
```

Comment:
This is a better set of instructions for what is in the developer's guide. Suggest we have only one and link to the source-of-truth.

Reply:
Counter argument for keeping them in dev guide: they are listed as discreet commands for people to reference, while this guide is specifically around release processes.